bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/GarbageCollectorThread.java

```diff
@@ -411,14 +415,20 @@ public void runWithFlags(boolean force, boolean suspendMajor, boolean suspendMin
         try {
             // gc inactive/deleted ledgers
             // this is used in extractMetaFromEntryLogs to calculate the usage of entry log
+            long gcLedgersStart = MathUtils.nowInNano();
             doGcLedgers();
```

Comment:
doGcLedgers may fail, can we record the metric in doGcLedgers method so that we can record the failed metrics?

Reply:
The same as extractMetaFromEntryLogs and doGcEntryLogs

Reply:
updated.